### ISSUE TITLE
Different mean_iou on VOC12

### ISSUE THREAD

**Issue by dingfg** [martinkersner/train-DeepLab#37]
I tested resnet-101 successfully, but it's not the same as the author's miou. The result before densecrf on the validation set was 0.7646 and the paper was 0.7635.
Strangely, the miou I ran out of VGG-16 was consistent with the paper.
Does anyone have the same problem?

**Comment by zhwis** [comment]
What did you use for evaluate? Is it evaluate_deeplab.py. I use VOCdevkit to evaluate result，but their value of iou is not the same. Thanks
